ledger/storage/leveldb/leveldb.go

```diff
@@ -92,16 +96,36 @@ func InitDB() (*LevelLedger, error) {
 	}
 
 	return &LevelLedger{
-		LDB: db,
+		ldb: db,
 	}, nil
 }
 
-// Get returns record from leveldb by timeslot and hash passed in record.Key
-func (ll *LevelLedger) Get(k record.Key) (rec record.Record, found bool) {
+// GetRecord returns record from leveldb by timeslot and hash passed in record.Key
+func (ll *LevelLedger) GetRecord(k record.Key) (rec record.Record, found bool) {
 	return nil, false
 }
 
-// Set stores record in leveldb
-func (ll *LevelLedger) Set(record record.Record) error {
+// SetRecord stores record in leveldb
+func (ll *LevelLedger) SetRecord(record record.Record) error {
 	return nil
 }
+
+// GetIndex fetches lifeline index from leveldb (records and lifeline indexes have the same id, but different scopes)
+func (ll *LevelLedger) GetIndex(id record.ID) (index record.LifelineIndex, found bool) {
```

Comment:
don't use named return values (some rare cases are ok, but this language feature is mostly for very short helper methods, and in production code, we should avoid it as much as possible)